_posts/2021-05-28-status-update.md

```diff
@@ -0,0 +1,105 @@
+
+1. Automatically publish any new release of a PowerShell module with DSC Resources on the gallery - either a new version or a new module - within 24 hours of release, straight to the [Forge][puppet-dsc-forge].
+2. Automatically rebuild and publish the incremented build of _all_ previously puppetized PowerShell modules with DSC Resources to the Forge.
+It's that second use case, the rebuilding, that makes this change in templating extremely effective:
+There are over `350` PowerShell modules with DSC Resources on the Gallery right now.
+Even if we only rebuilt the latest version of those modules, we would be saving almost _three hours_ of build time.
+
+But when we make updates to the core builder or pwshlib library, we're actually rebuilding older module versions, which means even if we are extremely conservative with our estimations and pretend the PowerShell modules only average 5 releases each, we're going to shave a _frankly astounding_ **875 hours** from our build time!
+
+The real time to rebuild and publish is, of course, shorter because we will be running this all in parallel; but saving more than a literal month's worth of build time on every release of `Puppet.Dsc` is an overwhelming win for us (and for the folks who maintain the Github Actions infrastructure)!
+
+[pdkgo-templating]: https://github.com/puppetlabs/pdkgo
+[puppet-dsc-autopublish]: https://github.com/puppetlabs/Puppet.Dsc/milestone/2
+[puppet-dsc-forge]: https://forge.puppet.com/modules/dsc
+[puppet-dsc-gallery]: https://www.powershellgallery.com/packages/Puppet.Dsc/0.5.0
+[puppet-dsc-intro-blog]: https://puppetlabs.github.io/iac/news/roadmap/2020/09/21/dsc-release.html
+[puppet-dsc-pdkgo-template]: https://github.com/puppetlabs/Puppet.Dsc/pull/154
+[puppet-dsc-stable]: https://github.com/puppetlabs/Puppet.Dsc/milestone/1
+
+
+### PDK Content Templates
+
+You may have seen the PDK user experience survey that went around some weeks ago.
+One of the hypothesis that we started with was a prediction that the monolithic [pdk-templates][pdk-templates] repository was too complex and inflexible for end users.
+The [results][pdk-results] of the survey bore this out, and so we started work on a new approach to content templating.
+Instead of a single repository which you'd need to fork and maintain to keep up to date, we built out a modular system that would allow you pick and choose the components you'd like to include.
+I'm happy to report that we are making great progress and will have a limited availability release out very soon for you to experiment with so watch this space for updates!
+
+[pdk-templates]: https://github.com/puppetlabs/pdk-templates
+[pdk-results]: https://docs.google.com/document/d/19LzpixC694nOklW6-IqwcGBzlwR6v56M-sMsdSS14Ss/edit#
```

Comment:
```suggestion
### Puppet Content Templates LA Release

You may have seen the PDK user experience survey that went around some weeks ago. We wanted to gain your feedback to validate some assumptions we had about creating content for Puppet products.  One of the hypothesis that we started with was a prediction that the monolithic [pdk-templates][pdk-templates] repository was too complex and inflexible for end users.

The [results][pdk-results] of the survey bore this out, and so we started work on a new approach to content templating.

Instead of a single repository which you'd need to fork and maintain to keep up to date, we built out a modular system that would allow you pick and choose the components you'd like to include.

We're happy to report that we are making great progress and will have a limited availability release out very soon for you to experiment with, so watch this space for updates!

[pdk-templates]: https://github.com/puppetlabs/pdk-templates
[pdk-results]: https://docs.google.com/document/d/19LzpixC694nOklW6-IqwcGBzlwR6v56M-sMsdSS14Ss/edit#
```

Reply:
It makes sense for this section to go before the `Puppet.Dsc` section as it explains a bit what PCT are before Lombardi starts using them